deployability/plugins/influxdb_reporter/README.md

```diff
@@ -0,0 +1,62 @@
+# Wazuh's InfluxDB plugin for pytest
+
+This is a plugin to send the results of the tests to an InfluxDB database.
+
+## Installation
+
+> This plugin is not available in PyPI yet. You can install it from source.
+
+1. Clone the repository
+    ```shellsession
+    $ git clone https://github.com/wazuh/wazuh-qa.git -b enhancement/4736-dtt1-influxdb-plugin
```

Comment:
Remove custom branch

Reply:
Done in [6311d92](https://github.com/wazuh/wazuh-qa/pull/5190/commits/6311d92724994e505494d41ca00dd7d9cddda76f)